usaspending_api/references/management/commands/loadtas.py

```diff
@@ -74,16 +88,12 @@ def handle(self, *args, **options):
         insert_federal_accounts()
 
     def generate_tas_rendering_label(self, row):
-        return TreasuryAppropriationAccount.generate_tas_rendering_label(row["ATA"],
-                                                                         row["Agency AID"],
-                                                                         row["A"],
-                                                                         row["BPOA"],
-                                                                         row["EPOA"],
-                                                                         row["MAIN"],
+        return TreasuryAppropriationAccount.generate_tas_rendering_label(row["ATA"], row["Agency AID"], row["A"],
+                                                                         row["BPOA"], row["EPOA"], row["MAIN"],
                                                                          row["SUB"])
 
     def skip_and_remove_financing_tas(self, row, instance):
-        if row["Financial Indicator Type2"] == "F":
+        if row["financial_indicator_type2"] == "F":
```

Comment:
I'm assuming the column name got lowered?

Reply:
Yes, the new files has it as `financial_indicator_type2`